DESCRIPTION

```diff
@@ -2,7 +2,7 @@ Package: compcodeR
 Type: Package
 Title: RNAseq data simulation, differential expression analysis and
     performance comparison of differential expression methods
-Version: 1.29.0
+Version: 1.29.0.9999
 Authors@R: c(person("Charlotte", "Soneson", role = c("aut", "cre"), 
```

Comment:
Please feel free to add yourselves as authors

Reply:
Added, thank you.